Desktop/modules/dynamicmodule.cpp

```diff
@@ -84,9 +84,9 @@ void DynamicModule::developmentModuleFolderCreate()
 }
 
 ///This constructor is meant specifically for the development module and only *it*!
-DynamicModule::DynamicModule(QObject * parent) : QObject(parent), _isDeveloperMod(true)
+DynamicModule::DynamicModule(QObject * parent, QString devFolder) : QObject(parent), _isDeveloperMod(true)
 {
-	_modulePackage	= Settings::value(Settings::DEVELOPER_FOLDER).toString().toStdString();
```

Comment:
Why did you do this?

Reply:
This is because for the generateWrapper start option, I use the installJASPDeveloperModule to load the module, so the folder is not in this case the Developer Module folder.

Reply:
but why?
why not use install a module?

Reply:
like, that function is quite aptly named "installJASPDeveloperModule" and that is expected to be in that one folder...

Now you change that but who knows which other checks could fail because of that?